samples/powerpoint/preview-apis/manage-hyperlinks.yaml

```
@@ -0,0 +1,74 @@
+id: powerpoint-manage-hyperlinks
+name: Get hyperlinks
+description: Gets hyperlinks found on a slide.
```

Nit: Let's favor saying "in" rather than "on" to match the object-oriented nature of the JS APIs.
```suggestion
description: Gets the hyperlinks found in a slide.
```